### PR TITLE
WDCZEN-9 feature 8.0: Allow running rocksdb MTR test cases on ZenFS storage in parallel

### PR DESCRIPTION
**Pull request by percona-ysorokin**
https://jira.percona.com/browse/WDCZEN-9

### CODE REVIEW

**Comment by percona-ysorokin** [comment]
Jenkins build link (based on custom ps-build scripts with zenfs --parallel extensions)
https://ps3.cd.percona.com/job/percona-server-8.0-pipeline-PS-7749/58/

File: mysql-test/lib/My/ConfigFactory.pm [hunk 1]
```diff
@@ -693,6 +695,9 @@ sub new_config {
     $hosts->{$host} = $baseport;
   }
 
+  # Worker ID
+  my $worker = $args->{'worker'};
```
Comment (oleksandr-kachan) [inline mysql-test/lib/My/ConfigFactory.pm]: It's used only in one place in this method. Shall we just pass it as &$rule($self, $config, $args->{'worker'}) below? Or maybe just declare it closer to the place where it is actually used.

File: mysql-test/mysql-test-run.pl [hunk 2]
```diff
@@ -4177,6 +4177,7 @@ sub default_mysqld {
                                     baseport      => 0,
                                     user          => $opt_user,
                                     password      => '',
+                                    worker        => 1,
```
Comment (oleksandr-kachan) [inline mysql-test/mysql-test-run.pl]: Just for better readability it may be better to add some constant for default worker ID. Up to you.

**Review by oleksandr-kachan — APPROVED** [review]
See some minor comments

**Review by oleksandr-kachan — APPROVED** [review]
LGTM

**Pull request merged**